docs/troubleshoot/cli/cli-use-curl-to-troubleshoot.md

```diff
@@ -60,10 +74,18 @@ For example, this bypasses SSL certificate verification for servers with self-si
 
 Required and displays as plain text. Also possible to [use an environment variable](../../user-guide/cli-using-using-environment-variables.md).
 
-- `<ID>`: Specifies the z/OSMF user identification.
-- `<PASSWORD>`: Specifies the user password for z/OSMF.
+- `<ID>`
+
+    Specifies the z/OSMF user identification.
+- `<PASSWORD>`
+
+    Specifies the user password for z/OSMF.
```

Comment:
Should we use consistent phrasing for both?
```suggestion
- `<ID>`

    Specifies the user ID for z/OSMF.
- `<PASSWORD>`
    Specifies the user password for z/OSMF.
```

Reply:
I prefer identification 😸